client/my-sites/site-settings/podcasting-details/publish-notice.jsx

```diff
@@ -0,0 +1,47 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import PodcastingSupportLink from './support-link';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getTerm } from 'state/terms/selectors';
+
+function PodcastingPublishNotice( { translate, podcastingCategoryName } ) {
+	return (
+		<div className="podcasting-details__publish-notice">
+			<Gridicon icon="microphone" size={ 24 } />
+			<span className="podcasting-details__publish-notice-text">
+				{ translate(
+					'Publish blog posts in the {{strong}}%s{{/strong}} category to add new episodes.',
+					{
+						args: podcastingCategoryName,
+						components: { strong: <strong /> },
+					}
+				) }
+				<PodcastingSupportLink />
+			</span>
+		</div>
+	);
+}
+
+export default connect( ( state, ownProps ) => {
+	if ( ownProps.podcastingCategoryId <= 0 ) {
+		return null;
+	}
+
+	const siteId = getSelectedSiteId( state );
+	const podcastingCategory = getTerm( state, siteId, 'category', ownProps.podcastingCategoryId );
+
+	return {
+		podcastingCategoryName: podcastingCategory && podcastingCategory.name,
```

Comment:
if either `podcastingCategory` or `podcastingCategory.name` are `undefined`/falsey, the value for `podcastingCategoryName` would be `false` here. I think this could cause visual issues in render:

> Publish blog posts in the **false** category to add new episodes.

Unless I've followed wrong.

If no `podcastingCategory` is provided do we assume some default category? Maybe we could echo that default here? If not perhaps some alternative copy should be used?